src/components/validations/with-validation.hoc.js

```diff
@@ -92,7 +96,11 @@ const withValidation = (WrappedComponent) => {
       return hasValidations;
     }
 
-    validate = (types = validationTypes, isOnSubmit) => {
+    checkContext(contextProp) {
+      return this.context && this.context[contextProp];
+    }
+
+    validate = (types = Object.keys(VALIDATION_TYPES), isOnSubmit) => {
```

Comment:
I think that `validate = (types = validationTypes, isOnSubmit) => {` was more readable

Reply:
Agreed again missed that it was included when I fixed the conflicts